tests/e2e/util.go

```diff
@@ -1633,13 +1637,71 @@ func DeleteStatefulPodAtIndex(client clientset.Interface, index int, ss *apps.St
 
 }
 
+//getClusterComputeResource returns the clusterComputeResource and vSANClient
+func getClusterComputeResource(ctx context.Context, vs *vSphere) ([]*object.ClusterComputeResource, *VsanClient) {
+	var err error
+	if clusterComputeResource == nil {
+		clusterComputeResource, vsanHealthClient, err = getClusterName(ctx, vs)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	}
+	return clusterComputeResource, vsanHealthClient
+}
+
+//findIP returns the IP from the input string
+func findIP(input string) string {
+	numBlock := "(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])"
+	regexPattern := numBlock + "\\." + numBlock + "\\." + numBlock + "\\." + numBlock
+
+	regEx := regexp.MustCompile(regexPattern)
+	return regEx.FindString(input)
+}
+
+//getHosts returns list of hosts and it takes clusterComputeResource as input
+//This method is used by WCP and GC tests
+func getHosts(ctx context.Context, clusterComputeResource []*object.ClusterComputeResource) []*object.HostSystem {
+	var err error
+	if hosts == nil {
+		computeCluster := os.Getenv(envComputerClusterName)
+		if computeCluster == "" {
+			if guestCluster {
+				computeCluster = "compute-cluster"
+			} else if supervisorCluster {
+				computeCluster = "wcp-app-platform-sanity-cluster"
+			}
+			framework.Logf("Default cluster is chosen for test")
```

Comment:
This line will be logged every time, right ? Shouldn't this be inside `else{}`

Reply:
This will be logged only if `os.Getenv(envComputeClusterName)` is not set, 
which means our test will run in the Default cluster which is deployed by our pipelines.